### ISSUE TITLE
DB recreation on each flutter hot restart

### ISSUE THREAD

**Issue by Viper-Bit** [isar/isar#453]
**Steps to Reproduce**
after each hot restart isar create a new db and removes all changes. i open isar in main method

```dart
void main() async {
  WidgetsFlutterBinding.ensureInitialized();

  var isar = await Isar.open([
    StudentSchema,
    TeacherSchema,
  ]);

  // do other things

  runApp(const MyApp());
}
```

**Version**
 - Platform: Windows
 - Flutter version: 3.1.0-9.0.pre
 - Isar version: 3.0.0-dev.3 | 3.0.0-dev.4
 - Isar Generator version: 3.0.0-dev.3 | 3.0.0-dev.4

**Comment by leisim** [comment]
Thank you this is a very important bug report. A fix is already on the way. @all-contributers please add @Viper-Bit for bug reports.

**Comment by leisim** [comment]
Thank you this is a very important bug report. A fix is already on the way. @all-contributors please add @Viper-Bit for bug reports.

**Comment by leisim** [comment]
Fixed in `3.0.0-dev.5`
